tutorials/fiori-ios-scpms-starter-mission-06/fiori-ios-scpms-starter-mission-06.md

```diff
@@ -21,14 +21,14 @@ time: 35
 
 ---
 
-[ACCORDION-BEGIN [Step 1: ](Implement the master-detail view)]
+[ACCORDION-BEGIN [Step 1: ](Implement a Split View)]
 
 In the last tutorial you made changes to your Xcode project settings to enable Mac Catalyst for your app project.
 This tutorial is about making some changes to the project to fit better on MacOS without changing the appearance on iPhone or iOS.
 
 We are going to cover the basics for Mac Catalyst and the SAP Cloud Platform SDK for iOS but using the previously mentioned documentation in combination with this tutorial series you should be well suited to build great business apps.
 
-In the beginning we want to change the app flow to have a split view in place when running on Mac Catalyst. The split view or Master-Detail View allows Mac Catalyst to display a sidebar similar to Apple's Music or Stocks applications on MacOS.
+In the beginning we want to change the app flow to have a split view in place when running on Mac Catalyst. The split view or Split View allows Mac Catalyst to display a sidebar similar to Apple's Music or Stocks applications on MacOS.
```

Comment:
Please don't say "we", say "you" instead as described in the [style guide](https://jam4.sapjam.com/wiki/show/MozDWaPczznL8NKeGEk0y9?_lightbox=true#mcetoc_1dtqgno1sd).